if_re.c

```diff
@@ -9047,29 +8825,6 @@ void			*xsc;
         return;
```

Comment:
I do not understand how to put a comment at line 8814.  But look at the spl block ending there, which should be removed.